fireflight/src/components/Register.jsx

```diff
@@ -89,7 +93,14 @@ function Register() {
           <Button type="submit" disabled={loading}>
             {loading ? "Loading..." : "Register"}
           </Button>
-          {badPassword ? <p>"Your passwords don't match"</p> : <></>}
+          {errorStatus ? (
+            <div>
+              <p>{errorText.username}</p>
+              <p>{errorText.password}</p>
+            </div>
```

Comment:
errorText is set as strings on passwords not matching, but here they are treated as there always being a class. This is going to crash on that error.